includes/class-wc-structured-data.php

```diff
@@ -268,7 +268,7 @@ public function generate_product_data( $product = null ) {
 			$markup['offers'] = array( apply_filters( 'woocommerce_structured_data_product_offer', $markup_offer, $product ) );
 		}
 
-		if ( $product->get_review_count() && wc_review_ratings_enabled() ) {
+		if ( $product->get_rating_count() && wc_review_ratings_enabled() ) {
```

Comment:
Seems like that you included some extra code in this PR, if you could please revert it.

Reply:
Hi, yes, this is an unrelated fix:

If rating is not mandatory on comments, generate_product_data() produces incorrect output.
For example items with comments but no reviews are output with rating of zero which is getting flagged in google webmasters tools under the heading "Rating is missing required best and/or worst values".

The output google is objecting to is eg:
"aggregateRating":{"@type":"AggregateRating","ratingValue":"0","reviewCount":1}}

postmeta in this case is:
_wc_average_rating 0
_wc_rating_count 0.0
_wc_review_count 1

product lookup table is reporting
rating_count 0
average_rating 0.00

the meta is correct and reflected in the product object, it is the test that is wrong since
if $product->get_review_count() 
may be true even if there are no ratings.
therefore the correct code should be to use get_rating_count() instead

Reply:
ok, removed from this pull